specs/ledger/hs/src/Ledger/Update.hs

```diff
@@ -421,7 +424,7 @@ instance STS UPVOTE where
             ) <- judgmentContext
         vts' <- trans @ADDVOTE $ TRC ((rups, dms), vts, vote)
         let pid = vote ^. vPropId
-        Set.size (Set.filter ((== pid) . fst) vts') < (pps ^. cfmThd) || pid `Set.member` (Map.keysSet cps) ?! LowerThanThdAndNotAlreadyConfirmed
+        Core.psSize (Set.singleton pid ◃ vts') < (pps ^. cfmThd) || pid `Set.member` (dom cps) ?! LowerThanThdAndNotAlreadyConfirmed
```

Comment:
This looks great! Some questions:

- If we'd use overloaded lists we would be able to write `[pid] ◃ vts'`. I don't know if that makes sense though.
- Can't we use `length` from `Data.Foldable` instead of `Core.psSize`?

Reply:
Yes, great idea!

Reply:
https://github.com/input-output-hk/fm-ledger-rules/issues/262